paradise_papers_django/search/urls.py

```diff
@@ -4,12 +4,12 @@
 from .views import (
     Index,
     ResultPage,
-    nodes,
+    NodeDetail,
 )
 
 
 urlpatterns = [
                   url(r'^$', Index.as_view(), name='index'),
                   url('lookup', ResultPage.as_view(), name='lookup'),
-                  url(r'^(?P<node_id>[0-9]+)$', nodes, name='nodes'),
+                  url(r'^(?P<slug>[\w-]+)/(?P<node_id>[0-9]+)$', NodeDetail.as_view(), name='nodes'),
```

Comment:
Old slug trick 😸 
But need to check if this is the best way to handle the details